src/README.md

```diff
@@ -24,6 +29,33 @@ You should see the following content if it runs correctly.
 Some api for developers has been added, check them out in route.py.
 The HTML templates are to represent the view of administrators.
 
+### Deploying on Heroku
+We will teach you how to use Heroku Cli to deploy the website
+First, create a heroku app and then connect it.
+```
+heroku create
+heroku git:remote -a "Your app name"
+```
+
+If you want to create a postgresql, the following command can help you.
+```
+heroku addons:create heroku-postgresql:hobby-dev
+```
+
+You should create a .ENV file includeing the following content.
+```
+DATABASE_URL="Your database url" or you will use sqlite as your database
+ENV="Staging" or "Production" or you will use the DevelopmentConfig
+```
+
+Push it to the heroku master branch.
+```
+git push heroku master
```

Comment:
Unfortunately I could not deploy it by these instructions. It would say `remote:  !     No default language could be detected for this app.` after `git push heroku master`. Did you test it by yourself?

Reply:
Also, the `heroku create` command would change the upsteam of branch heroku. You may want to change this branch to another name?

Reply:
I use 
git push heroku heroku:master 
to run
But I think I should write  "git push heroku master" on the readme